pages/common/git-hash-object.md

```diff
@@ -0,0 +1,20 @@
+# git hash-object
+
+> Computes the unique hash key of content and optionally creates an object with specified type.
+> More information: <https://git-scm.com/docs/git-hash-object>.
+
+- Compute the ID that would be used to store it in git database:
```

Comment:
```suggestion
- Compute the ID used to store it in the git database:
```
I think this would be better.

Reply:
I wanted to express that it isn't stored in the git database, but returns the value that would be stored if we wanted it (by providing -w option).

Reply:
I think a simple English is better here
```suggestion
- Compute the object ID without store it:
```